docs/MigrationGuide.md

```diff
@@ -0,0 +1,40 @@
+# Migration Guide
+
+## Migration from v2 to v3
+
+Some changes can be automatically migrated from v2:
+
+Load manually the package `Stargate-Deprecated-V3`. It will include deprecations and some of them includes automatic rewrite expressions.
```

Comment:
Load manually --> Manually load
It will include deprecations and some of them includes automatic rewrite expressions. --> It includes deprecations, some of them with automatic rewrite expressions.